gordo_components/builder/build_model.py

```diff
@@ -214,12 +214,15 @@ def calculate_model_key(
     # Sets a lot of the parameters to json.dumps explicitly to ensure that we get
     # consistent hash-values even if json.dumps changes their default values (and as such might
     # generate different json which again gives different hash)
+    major, minor, _path = _parse_version(__version__)
```

Comment:
`_path` or `_patch`

Reply:
ha, misspelled that every time I wrote it, this one slipped by. Thanks!